pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -148,6 +148,7 @@
   company_strongnet: Strongsville School District
   company_TukwilaSchoolDistrict: Tukwila School District
   company_vermilionschools: Vermilion Local School District
+  company_vista-equity-partners: "Vista Equity Partners"
```

Comment:
Also is it consistent to use dashes when the others seem to be camel case or all lower-case?